.eslintrc.js

```diff
@@ -1,6 +1,65 @@
-const path = require('path');
-const Neutrino = require('neutrino');
-
-const api = new Neutrino([path.resolve('./neutrino-custom/development.js')]);
-
-module.exports = api.custom.eslintrc();
+module.exports = {
+  root: true,
+  extends: 'eslint-config-airbnb',
+  parser: 'babel-eslint',
+  settings: {
+    react: {
+      version: '16.6',
+    },
+  },
+  env: {
+    browser: true,
+    jasmine: true,
+  },
+  rules: {
```

Comment:
There are more rule-disablings required here than before, since the newer ESLint, eslint plugins (for react/imports/...) and AirBnB configs now check for a lot more issues than before by default. I'd like to enable as many of these as possible soon.